content/en/contribute/code/workflow.md

```diff
@@ -251,21 +253,38 @@ The ticket needs further development.
 {{< /tab >}}
 {{< /tabpane >}}
 
-A great way to facilitate discussion and collaboration is with a Draft PR.
+#### Create a branch
+
+Any code should be in a branch in each repository you update. The name of the branch should be in the form `<issue-number>-<readable-name>`, for example, `1104-inclusive-export`. 
+
+Push commits at least once a day to a remote repository, ensures that the code is always backed up and safe, protects against accidental deletes and allows team members to see the latest changes and work together more effectively.
+
+#### Pull requests
+
+Create a Draft Pull Request to facilitate discussion and collaboration with quality assistance engineers and developers.
+
+Once you are confident that the change is complete and ready to be merged:
+
+1. Change the Pull Request from `Draft` to `Ready for review`.
+2. The Pull Request title will be the commit message, it is important to follow the [commit message format]({{< ref "#commit-message-format" >}}) to name the Pull Request title properly. 
+3. Add a Pull Request description:
 
-Once you're confident that the change is complete and ready to be merged:
+   - Add a description of changes, decisions, backstory, and any extra information to facilitate review process and reduce follow-ups. 
```

Comment:
This is overlapping a bit with the `Updating The Issue With What You Actually Did` section. I think we can merge them. I will write a suggestion in the final PR review comment.

Reply:
Similar, but with a different audience, the PR usually contains more details for the reviewer, e.g the thinking process.
But I didn't explain it well here.